polaris-react/src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -74,3 +90,12 @@
 .ActionTitle-disabled {
   color: var(--p-color-text-interactive-disabled);
 }
+
+// The markup that uses theis className Only exists for polaris-summer-editions
```

Comment:
Not behind an `#{$se23}` flag because this class is specifically applied only to markup in `FileUpload` that is rendered when the `polarisSummerEditios2023` flag is enabled in React.